plezanje-net/web
# Home auth &more

Fetch data of home components again when user auth status changes. 

Test this with plezanje-net/api#88

To test:
- Exposed warnings
move some dates so that warnings are shown.
change referenced crag by a warning to point to a hidden crag
change referenced route by another warning to point to a route that belongs to a hidden crag
Observe display of warnings on home while logging in/out.

- Popular crags
Tick some routes of a hidden crag.
Observe display of popular crags on home while logging in/out.

- Latest images
Make a few of the latest images reference a hidden crag and/or a route belonging to a hidden crag.
Observe display of images on home while logging in/out.

- Latest ticks
Tick some routes of a hidden crag.
Observe display of latest ticks on home while logging in/out.

See commit messages for some other self explanatory minor changes.

closes #212
